tests/viz/test_exporter.py

```diff
@@ -182,19 +186,20 @@ def test_import_state_from_vtu_single_subdomains(
     # Define keys (here corresponding to all data stored in the vtu file to pass the
     # test).
     keys = ["dummy_scalar", "dummy_vector"]
+    keys_pts = ["dummy_scalar_pt", "dummy_vector_pt"]
```

Comment:
See my reminder above. I would suggest having a test, which uses the same key for cell and point data to test for expected behavior.